src/media/css/account.styl

```diff
@@ -81,6 +81,10 @@
         display: none;
     }
 }
+.html-rtl .region span {
```

Comment:
That does sound like a nice pattern, but it might not be the best for us, especially given how all our CSS is structured currently. I think it's more likely we'd want to modify a specific representation of an element rather than wanting to see the whole thing. In this case, we want to focus on the RTL portions of elements. Or for desktop refresh, we wanted to work with just the responsiveness portions of elements.